client/src/components/leftSidebar/infoMenu.js

```diff
@@ -0,0 +1,74 @@
+// jshint esversion: 6
+import React from "react";
+import { Button, Menu, MenuItem, Popover, Position } from "@blueprintjs/core";
+import { IconNames } from "@blueprintjs/icons";
+
+const InformationMenu = React.memo((props) => {
+  const { libraryVersions, tosURL, privacyURL } = props;
+  return (
+    <Popover
+      content={
+        <Menu>
+          <MenuItem
+            href="https://chanzuckerberg.github.io/cellxgene/"
+            target="_blank"
+            icon="book"
+            text="Documentation"
+            rel="noopener"
+          />
+          <MenuItem
+            href="https://join-cellxgene-users.herokuapp.com/"
+            target="_blank"
+            icon="chat"
+            text="Chat"
+            rel="noopener"
+          />
+          <MenuItem
+            href="https://github.com/chanzuckerberg/cellxgene"
+            target="_blank"
+            icon="git-branch"
+            text="Github"
+            rel="noopener"
+          />
+          <MenuItem
+            target="_blank"
+            text={
+              libraryVersions && libraryVersions.cellxgene
+                ? libraryVersions.cellxgene
+                : null
+            }
+          />
+          <MenuItem text="MIT License" />
+          {tosURL ? (
```

Comment:
I think we don't need ternary if the alternative is `null`!

E.g., `{ tosURL && <MenuItem href={tosURL} target="_blank" text="Terms of Service" /> }`